shipyard.md

```diff
@@ -11,27 +11,44 @@ A shipyard file can consist of any number of stages. A stage has the following p
 
 * **Name**. The name of the stage. This name will be used for the GitOps branch and the Kubernetes namespace to which services at this stage will be deployed to. Future versions will allow to explicitly define the target namespace.
 
-* **Deployment Strategy**. The deployment strategy used to deploy a new version of a service. Keptn supports deployment strategies of type: 
+* **Deployment Strategy**. The deployment strategy defines how an artifact is deployed. Currently, the following strategies are supported: 
   * `direct`: Deploys a new version of a service by replacing the old version of the service.
   * `blue_green_service`: Deploys a new version of a service next to the old one. After a successful validation of this new version, it replaces the old one and is marked as stable (i.e. it becomes the `primary`-version).
 
-   Future versions of Keptn will also support canary deployments.
+   Future versions of Keptn will also support canary releases.
 
-* **Test Strategy**. Defines the test strategy used to validate a deployment. Failed tests result in an automatic roll-back of the latest deployment in case of a blue/green deployment strategy. Keptn supports tests of type:
+* **Test Strategy**. Defines the test strategy used to validate a deployment. Failed tests result in an automatic roll-back of the latest deployment in case of a blue/green deployment strategy is used. Keptn supports tests of type:
   * `functional` 
   * `performance` 
 
+  In future versions of Keptn, it is not necessary to distinguish between functional and performance tests. But it is necessary to define multiple tests for a stage that can either be executed sequentially or in parallel. Therefore, we need a link between the shipyard and the uniform (i.e. using selectors and labels as known by k8s).
```

Comment:
I think we can get rid of the "In future version ..." paragraphs in the entire document. We propose a spec change. What future version might support would again be a spec change. Let's try to keep spec changes clean and focused.

Reply:
I agree. A spec should reflect the current implementation. 
(To my opinion: Docs in the ./design-docs can look into the future and can contain feature request.)

Reply:
In a recent discussion, we found that it might make sense to separate the specification from design-docs and RFCs. We might create a new repository for those artifacts to keep the spec repository clean.